spring-shell2-shell1-adapter/src/main/java/org/springframework/shell2/legacy/LegacyParameterResolver.java

```diff
@@ -41,10 +42,18 @@
  * Resolves parameters by looking at the {@link CliOption} annotation and acting accordingly.
  *
  * @author Eric Bottard
+ * @author Camilo Gonzalez
  */
 @Component
 public class LegacyParameterResolver implements ParameterResolver {
 
+	private static final String CLI_OPTION_NULL = "__NULL__";
+	
+	/**
+	 * Prefix used by Spring Shell 1 for the argument keys (e.g. command --key value)
```

Comment:
Missing dot at end of sentence, and __e.g.__ should be enclosed in `<em></em>` to not confuse the javadoc parser